scalameter-core/src/main/scala/org/scalameter/PicklerBasedKey.scala

```diff
@@ -0,0 +1,15 @@
+package org.scalameter
+
+import org.scalameter.picklers.Pickler
+
+
+/* A mixin for keys whose values require pickler based serialization. */
+trait PicklerBasedKey[T] {
```

Comment:
Ah, of course - it's a case class, I missed that.
I guess it's still a good idea to make `PicklerBasedKey` serializable.